React/books/src/app/components/Book/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import actionsCreators from '../../../redux/Book/actions';
+
+class Book extends Component {
+  componentDidMount = () => {
+    this.getBooks();
+  };
+
+  getBooks = () => {
+    this.props.dispatch(actionsCreators.getBooks());
+  };
+
+  renderBooks = () => {
+    if (this.props.listBooks) {
+      return this.props.listBooks.map(book => <li key={book.id}>{`${book.title} ${book.author}`}</li>);
+    }
+  };
+
+  renderStateRequest() {
+    if (this.props.loading && !this.props.failure) {
+      return 'Cargando...';
+    } else if (this.props.loading && this.props.failure) {
+      return 'Error al cargar libros';
+    }
+    return 'Books:';
+  }
+  renderSequest() {
+    if (this.props.loading && !this.props.failure) {
+      return false;
+    } else if (this.props.loading && this.props.failure) {
+      return true;
+    }
+    return true;
```

Comment:
It's generally not necessary to return booleans when comparing other booleans.
Here you return false only when `this.props.loading && !this.props.failure`. So you just returning `!(this.props.loading && !this.props.failure)` should be the same.

Anyway, check the comment about `loading: true`, you won't need some of this code.